<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="main_window_of_app">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1151</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="win_img">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="action_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Давай поиграем, нажми начать игру</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="screen_of_game">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="input_choice"/>
      </item>
      <item>
       <widget class="QPushButton" name="guess_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Угадать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="cow_img">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>cow_img</string>
     </property>
    </widget>
    <widget class="QLabel" name="bull_img">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>bull_img</string>
     </property>
    </widget>
    <widget class="QLabel" name="X_img">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="X_img2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="bull_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 28pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="cow_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 28pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>341</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="title">
     <string>Ход игры</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>92</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>92</number>
     </attribute>
     <column>
      <property name="text">
       <string>Число</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Быки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Коровы</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="title">
     <string>Настройки игры</string>
    </property>
    <widget class="QComboBox" name="difficulty_select">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Легкий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Средний</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сложный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Очень сложный</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="digit_capacity_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Кол-во цифр:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Вариация чисел:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_game_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);
font: 16pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Начать игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="finish_game_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);
font: 16pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Завершить игру</string>
     </property>
    </widget>
    <widget class="QLabel" name="variable_numbers_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1, 2, 3, 4</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="history_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
font: 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>История фермера</string>
    </property>
   </widget>
   <widget class="QPushButton" name="how_to_play_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
font: 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Забыл как играть?</string>
    </property>
   </widget>
   <zorder>win_img</zorder>
   <zorder>screen_of_game</zorder>
   <zorder>action_label</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>history_btn</zorder>
   <zorder>how_to_play_btn</zorder>
  </widget>
  <widget class="QLabel" name="background_game">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1191</width>
     <height>661</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background_game</zorder>
  <zorder>main_window_of_app</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
